dhis-2/dhis-services/dhis-service-dxf2/src/main/java/org/hisp/dhis/dxf2/metadata/DefaultMetadataImportService.java

```diff
@@ -184,6 +184,11 @@ public MetadataImportParams getParamsFromMap(Map<String, List<String>> parameter
     params.setFirstRowIsHeader(getBooleanWithDefault(parameters, "firstRowIsHeader", true));
     params.setAsync(getBooleanWithDefault(parameters, "async", false));
 
+    if (params.getMergeMode() == MergeMode.MERGE) {
+      throw new MetadataImportException(
+          "Merge mode MERGE is no longer supported, only merge mode REPLACE is available.");
```

Comment:
This is at the request parameters validation process in the controller, in which we usually throw direct exception. Then those exception will be caught by the `CrudControllerAdvice`. I'm not sure if we agree to always use `ErrorCode` at this stage. We do throw ErrorCode when in metadata import process. 
I think the whole error catching of `api/metadata` endpoint need some refactoring including exceptions thrown by database layer always result in http error 500.
So i think this could be another refactoring PR, will discuss with team.